### DISCUSSION TITLE
clang-cl toolchain's error message is incorrect

### DISCUSSION THREAD

**Discussion by MaxwellGengYF** [xmake-io/xmake#3814] [Ideas]
### Xmake Version

2.7.9

### Operating System Version and Architecture

Windows

### Describe Bug

when we call:
> xmake f --toolchain=clang-cl
but LLVM sdk's path is not inside PATH environment, the error message is:
> checking for Microsoft Visual Studio (x64) version ... no

### Expected Behavior

Somehow this error message should be like:
> checking for clang-cl ... no

**Comment by waruqi** [comment]
这个改不了，你没切平台，默认全局 msvc 工具链总归会去检测的，但这并不影响使用。。至于你指定的 `--toolchain=clang-cl`

会在实际 `xmake` build 阶段，根据每个 target 生效的真实 toolchain 延迟检测。。

> checking for clang-cl ... no

也就是说这个检测是 build 阶段才会去做，`xmake f` 阶段，其实还没去做 clang-cl 的检测，还没用到它。
